### DISCUSSION TITLE
Bug: Create new popover got black background

### DISCUSSION THREAD

**Discussion by omar2205** [community/community#60138] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

As you can see, there is a black rectangle around the popover which doesn't have the correct border-radius
![image](https://github.com/community/community/assets/1373867/bd2fdeb3-db52-4070-8538-8b92276d793a)

**Comment by CodeMoreira** [comment]
Can you show the full code of this select and dropdown? It's probably happening because of some wrong style

**Comment by DSDmark** [comment]
Hi @omar2205,

it is due to that class **Overlay**.

![image](https://github.com/community/community/assets/85744816/ed3453b1-4f39-4d4e-a6ae-1e08267c80db)

![image](https://github.com/community/community/assets/85744816/3dff07c5-5fc8-49da-b2dd-7b944851669a)
